src/js/common/schemaform/FormApp.jsx

```diff
@@ -62,15 +63,16 @@ export default class FormApp extends React.Component {
 
 
     return (
-      <div className="row">
-        <Element name="topScrollElement"/>
-        <div className="usa-width-two-thirds medium-8 columns">
-          {formConfig.title && !isIntroductionPage && <FormTitle title={formConfig.title} subTitle={formConfig.subTitle}/>}
-          {content}
-        </div>
-        <div className="usa-width-one-third medium-4 columns show-for-medium-up">
+      <div>
+        <div className="row">
+          <Element name="topScrollElement"/>
+          <div className="usa-width-two-thirds medium-8 columns">
+            {formConfig.title && !isIntroductionPage && <FormTitle title={formConfig.title} subTitle={formConfig.subTitle}/>}
+            {content}
+          </div>
+          <span className="js-test-location hidden" data-location={trimmedPathname} hidden></span>
         </div>
-        <span className="js-test-location hidden" data-location={trimmedPathname} hidden></span>
```

Comment:
@jbalboni, this is actually on line 73 now, unless you meant that is should be moved back down.